Directory.Build.props

```diff
@@ -2,8 +2,10 @@
   <PropertyGroup>
     <RepositoryName>wpf</RepositoryName>
     <WindowsDesktopARM64Support>true</WindowsDesktopARM64Support>
-    <TargetFramework>net8.0</TargetFramework>
-    <TargetFrameworkVersion>8.0</TargetFrameworkVersion>
+    <TargetFramework>net481</TargetFramework>
```

Comment:
We'll want to do this using `<TargetFrameworks>net8.0;net481</TargetFrameworks>`